doc/2/api/protocols/http/index.md

```diff
@@ -64,3 +64,23 @@ The listening port can be modified under the `server.port` section of the [confi
 HTTP and WebSocket protocols share the same underlying server instance.  
 Modifying the listening port will impact these two protocols.
 :::
+
+### HTTP-only options
+
+Only the HTTP protocol does support the following options in a [KuzzleRequest](/core/2/api/payloads/request)
+
+| Property     | Type               | Description                                                                           |
+| ------------ | ------------------ | ------------------------------------------------------------------------------------- |
+| `cookieOnly` | <pre>boolean</pre> | Instructs Kuzzle to store / use authentication tokens within secured HTTP Cookies     |
```

Comment:
Sorry for mentioning this only now, but after thinking about it, I don't like the name `cookieOnly`, I don't find it self-explaining.

What about somethiing like `cookieAuth` or something like that? (open to debate obviously)

Reply:
Yep, I like this idea !
Something like `cookieAuth` makes sense, `cookieOnly` makes me think that _sometimes_ cookie **and** sent tokens could be allowed at the same time?